elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -1207,7 +1207,10 @@ elytron.credential-store.loaded-provider.version=The version of the provider use
 elytron.credential-store.providers=The name of the providers defined within the subsystem to obtain the Providers to search for the one that can create the required CredentialStore type. If this is not specified then the global list of Providers is used instead.
 elytron.credential-store.other-providers=The name of the providers defined within the subsystem to obtain the Providers to search for the one that can create the required JCA objects within credential store. This is valid only for key-store based CredentialStore. If this is not specified then the global list of Providers is used instead.
 elytron.credential-store.provider-name=The name of the provider to use to instantiate the CredentialStoreSpi. If the provider is not specified then the first provider found that can create an instance of the specified 'type' will be used.
-elytron.credential-store.uri=The URI for credential store definition.
+elytron.credential-store.implementation-properties=Map of credentials store implementation specific properties.
+elytron.credential-store.location=File name of credential store storage.
+elytron.credential-store.modifiable=Specifies whether credential store is modifiable (true | false).
```

Comment:
true/false not really needed - it is covered by the type=boolean in :read-resource-description